ICSharpCode.Decompiler/IL/Instructions.tt

```diff
@@ -265,7 +265,7 @@
 		new OpCode("default.value", "Returns the default value for a type.",
 			NoArguments, HasTypeOperand, ResultType("type.GetStackType()")),
 		new OpCode("throw", "Throws an exception.",
-			Unary, MayThrow, UnconditionalBranch),
+			Unary, MayThrow, HasFlag("InstructionFlags.EndPointUnreachable"), ResultType("this.resultType")),
```

Comment:
Why did we replace `UnconditionalBranch` with just `EndPointUnreachable` here?
The `rethrow` instruction still uses `UnconditionalBranch`, which seems inconsistent to me.

Reply:
`UnconditionalBranch = VoidResult + HasFlag("InstructionFlags.EndPointUnreachable")`, so this makes sense.